alf/environments/suite_unittest_test.py

```diff
@@ -16,23 +16,30 @@
 import numpy as np
 
 import unittest
+from absl.testing import parameterized
 import tensorflow as tf
 
+from alf.environments.suite_unittest import ActionType
 from alf.environments.suite_unittest import ValueUnittestEnv
 from alf.environments.suite_unittest import PolicyUnittestEnv
 from alf.environments.suite_unittest import RNNPolicyUnittestEnv
 from tf_agents.trajectories.time_step import TimeStep, StepType
 
 
-class SuiteUnittestEnvTest(unittest.TestCase):
+class SuiteUnittestEnvTest(parameterized.TestCase, unittest.TestCase):
     def assertArrayEqual(self, x, y):
         self.assertEqual(x.shape, y.shape)
         self.assertEqual(float(tf.reduce_max(abs(x - y))), 0.)
 
-    def test_value_unittest_env(self):
+    @parameterized.parameters(False, True)
```

Comment:
It is simpler and easier to read to use Action.Discrete and Action.Continuous as the parameter.